src/modules/workspaces/workspaces.usecase.spec.ts

```diff
@@ -299,6 +300,51 @@ describe('WorkspacesUsecases', () => {
     });
   });
 
+  describe('editWorkspaceDetails', () => {
+    const user = newUser();
+    const workspace = newWorkspace({ owner: user });
+    const editWorkspaceDto: EditWorkspaceDetailsDto = {
+      name: 'Test Workspace',
+      description: 'Workspace description',
+    };
+    it('When workspace does not exist, then it should throw', async () => {
+      jest.spyOn(workspaceRepository, 'findById').mockResolvedValueOnce(null);
+
+      await expect(
+        service.editWorkspaceDetails('workspace-id', user, editWorkspaceDto),
+      ).rejects.toThrow(NotFoundException);
+    });
+
+    it('When user is not the owner of the workspace, then it should throw', async () => {
+      jest
+        .spyOn(workspaceRepository, 'findById')
+        .mockResolvedValueOnce(workspace);
+
+      await expect(
+        service.editWorkspaceDetails(
+          'workspace-id',
+          newUser(),
+          editWorkspaceDto,
+        ),
+      ).rejects.toThrow(ForbiddenException);
+    });
+
+    it('When workspace is valid, then it should update the workspace details', async () => {
```

Comment:
_When the user is the owner of the workspace, then it should update the workspace details_

Seems to be more accurate